Mlem/Views/Tabs/Settings/Components/Views/General.swift

```diff
@@ -92,6 +93,30 @@ struct GeneralSettingsView: View
                 }
 
             }
+
+            Section()
+            {
+                Button(role: .destructive) {
+                    URLCache.shared.removeAllCachedResponses()
+                    diskUsage = Int64(URLCache.shared.currentDiskUsage)
+                } label: {
+                    Label("Image cache: \(ByteCountFormatter.string(fromByteCount: diskUsage, countStyle: .file))", systemImage: "trash")
```

Comment:
should we just refer to this as `Cache`, although we're explicitly using it for images we're using the systems `.shared` instance, so it will be storing more than images potentially

Reply:
Yea but the idea was that maybe we'd want to have more then one cache (think image cache and videos cache so a user can clear the massive video cache but not have to suffer all the images reloading as well)

but for now I've updated it

Reply:
> Yea but the idea was that maybe we'd want to have more then one cache

Yeah, my thinking too - but while we have one it's best to keep the wording generic 👍